curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -277,7 +277,18 @@ void FuseS3Client::WarmUpAllObjs(
                 delete []context->buf;
                 return;
             }
-            // todo: retry
+            if (context->retry >= MAX_RETRY_TIME) {
+                if (pendingReq.fetch_sub(1, std::memory_order_seq_cst) == 1) {
+                    VLOG(6) << "pendingReq is over";
+                    cond.Signal();
+                } else {
+                    LOG(WARNING) << "Up to max retry times";
+                }
+                delete []context->buf;
+                return;
```

Comment:
WarmUpAllObjs may be failed, should add return status at WarmUpAllObjs 

It is necessary to judge the return value of WarmUpAllObjs at the place where WarmUpAllObjs is called

Reply:
> WarmUpAllObjs may be failed, should add return status at WarmUpAllObjs
> 
> It is necessary to judge the return value of WarmUpAllObjs at the place where WarmUpAllObjs is called

there is no need to do that, because Warm-up is just a matter of downloading as much as possible, not a necessity